projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectPreview.java

```diff
@@ -0,0 +1,66 @@
+package life.qbic.projectmanagement.domain.project;
+
+import static java.util.Objects.requireNonNull;
+
+
+import javax.persistence.*;
+import java.util.Objects;
+
+/**
+ * A limited view of the more complex {@link Project}.
+ *
+ * @since 1.0.0
+ */
+@Entity
+@Table(name = "projects_datamanager")
+public class ProjectPreview {
```

Comment:
Wasn't there another class doing the same?

Reply:
please see the class' JavaDocs :)